src/access_nri_intake/catalog/manager.py

```diff
@@ -47,13 +48,16 @@ def __init__(self, path: str):
 
         self.mode = "a" if os.path.exists(path) else "w"
 
-        self.dfcat = DfFileCatalog(
-            path=self.path,
-            yaml_column=YAML_COLUMN,
-            name_column=NAME_COLUMN,
-            mode=self.mode,
-            columns_with_iterables=COLUMNS_WITH_ITERABLES,
-        )
+        try:
+            self.dfcat = DfFileCatalog(
+                path=self.path,
+                yaml_column=YAML_COLUMN,
+                name_column=NAME_COLUMN,
+                mode=self.mode,
+                columns_with_iterables=COLUMNS_WITH_ITERABLES,
+            )
+        except EmptyDataError as e:
+            raise EmptyDataError(str(e) + f": {self.path}")
```

Comment:
Perhaps an issue for intake-dataframe-catalog rather than here, but I feel like we might want to emit a DfFileCatalogError here rather than than an EmptyDataError?

Reply:
I think that is an issue for `intake-dataframe-catalog`, as you suggested. All I was trying to do is re-emit the same error with a slightly more useful message.

Reply:
I've opened an issue there. Could you update the catch to 
```python
except (EmptyDataError, DfFileCatalogError) as e
```
so it won't break when we update it there?